importer/yarn_parser.gd

```diff
@@ -137,3 +120,19 @@ static func parse_body(file: File, indent_level = 0):
 			body.append(node)
 
 	return body
+
+
+static func parse_command_args(args : String):
+	# Regex to match subquotes: 
+	# https://stackoverflow.com/questions/2817646/javascript-split-string-on-space-or-on-quotes-to-array
+	var commandRegex = '[^\\s"]+|"[^"]+"'
+	var params = []
+	var regex = RegEx.new()
+
+	regex.compile(commandRegex)
+
+	for param in regex.search_all(args):
+		params.push_back(param.get_string() as String)
+
+	return params
```

Comment:
@Technohacker I think the original intent of this was just to allow subquote strings? The regex seemed to work a bit more cleanly and will hopefully prevent edge cases. ... but let me know if I missed something.

Reply:
If the regex works, then yes it's a much cleaner solution. I guess I'll add in a little explanation of what the RegEx pattern is doing for documentation purposes :)

Reply:
It worked in limited testing. I guess that's "robust enough" for now? If further issues are opened, I would say it can be looked into again, but my instinct is it will probably hold up a bit better.